content/en/docs/concepts/workloads/controllers/garbage-collection.md

```diff
@@ -5,180 +5,137 @@ weight: 60
-
-```shell
-kubectl apply -f https://k8s.io/examples/controllers/replicaset.yaml
-kubectl get pods --output=yaml
-```
-
-The output shows that the Pod owner is a ReplicaSet named `my-repset`:
-
-```yaml
-apiVersion: v1
-kind: Pod
-metadata:
-  ...
-  ownerReferences:
-  - apiVersion: apps/v1
-    controller: true
-    blockOwnerDeletion: true
-    kind: ReplicaSet
-    name: my-repset
-    uid: d9607e19-f88f-11e6-a518-42010a800195
-  ...
-```
-
-{{< note >}}
-Cross-namespace owner references are disallowed by design.
-
-Namespaced dependents can specify cluster-scoped or namespaced owners.
-A namespaced owner **must** exist in the same namespace as the dependent.
-If it does not, the owner reference is treated as absent, and the dependent
-is subject to deletion once all owners are verified absent.
-
-Cluster-scoped dependents can only specify cluster-scoped owners.
-In v1.20+, if a cluster-scoped dependent specifies a namespaced kind as an owner,
-it is treated as having an unresolvable owner reference, and is not able to be garbage collected.
-
-In v1.20+, if the garbage collector detects an invalid cross-namespace `ownerReference`,
-or a cluster-scoped dependent with an `ownerReference` referencing a namespaced kind, a warning Event 
-with a reason of `OwnerRefInvalidNamespace` and an `involvedObject` of the invalid dependent is reported.
-You can check for that kind of Event by running
-`kubectl get events -A --field-selector=reason=OwnerRefInvalidNamespace`.
-{{< /note >}}
-
-## Controlling how the garbage collector deletes dependents
-
-When you delete an object, you can specify whether the object's dependents are
-also deleted automatically. Deleting dependents automatically is called *cascading
-deletion*.  There are two modes of *cascading deletion*: *background* and *foreground*.
-
-If you delete an object without deleting its dependents
-automatically, the dependents are said to be *orphaned*.
+{{<glossary_definition term_id="garbage-collection" length="short">}} This
+allows the clean up of resources like the following:
+
+  * [Objects without owner references](#owners-dependents)
+  * [Unused containers and container images](#containers-images)
+  * [Failed pods](/docs/concepts/workloads/pods/pod-lifecycle/#pod-garbage-collection)
+  * [Dynamically provisioned PersistentVolumes with a StorageClass reclaim policy of Delete](/docs/concepts/storage/persistent-volumes/#delete)
+  * [Stale or expired CertificateSigningRequests (CSRs)](/reference/access-authn-authz/certificate-signing-requests/#request-signing-process)
+  * Nodes deleted in the following scenarios:
+    * On a cloud when the cluster uses a [cloud controller manager](/docs/concepts/architecture/cloud-controller/)
+    * On-premises when the cluster uses an addon similar to a cloud controller
+      manager
+  * [Node Lease objects](/docs/concepts/architecture/nodes/#heartbeats)
+  * [Completed Jobs](/docs/concepts/workloads/controllers/ttlafterfinished/)
```

Comment:
As this is in currently part of workloads section, _this_ page should either:
- focus on cleanup for completed jobs
- move somewhere else

Reply:
By this page, do you mean the TTLafterfinished page?